### ISSUE TITLE
[Feature]:  More list options (auto tags, auto dates...)

### ISSUE THREAD

**Issue by webenadam** [mgmeyers/obsidian-kanban#993]
### Goal or desired outcome of this feature

Making the kanban more automatic and flexible for different use cases

### Describe the feature

Adding more options to list (per list settings). for example:
- Auto add specific tag to cards on this list
- Auto add specific date to cards on this list (could be used for sort of a calander on a kanban)

### Can you think of any alternatives or work-arounds?

of course you can do it your self, but you could also cut a tree and make a paper note :)

**Comment by Kiryoko** [comment]
This would be awesome!

Being able to auto-add specific tags to cards on a list upon creating a new one is the only thing I'm missing from this plugin :)

**Comment by sa-nny** [comment]
I would also love to have this auto-add feature because when I'm working with dateview I want to show a dynamic timeline based on dates or tags. Currently I found out that kanban can't use custom templates.

**Comment by Dexmach5723** [comment]
I agree with this. This would make the connection to third-party apps like Morgen possible. Morgen only uses its AI based on tags so I have to use another Kanban plugin that tags the tasks instead of using headers. I much prefer this plugin, but Morgen has become too much a part of my workflow that I have abandoned this plugin for the Task Kanban plugin (which is very basic and no longer updated)
